### PR TITLE
Add post about the pathogen detection grant from EOSC

### PR DESCRIPTION
**Pull request by bebatut**

### CODE REVIEW

File: _posts/2021-12-08-pathogen-detection-eosc-life-grant.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+site: freiburg
+tags:
+title:
+  Accessible and scalable detection and identification of foodborne pathogens, a
+  project with Biolytix, a swiss company, and funded by EOSC-Life
```
Comment (bgruening) [inline _posts/2021-12-08-pathogen-detection-eosc-life-grant.md:6]: I would remove the a Swiss company from the title

**Review by bgruening — APPROVED** [review]
Just one small comment

File: _posts/2021-12-08-pathogen-detection-eosc-life-grant.md (hunk 1 above)
Comment (bebatut) [inline _posts/2021-12-08-pathogen-detection-eosc-life-grant.md:6]:
```suggestion
  project with Biolytix and funded by EOSC-Life
```

**Review by beatrizserrano — APPROVED** [review]
Thanks for the post and congrats! 🎉

**Pull request merged**